src/components/file-input/file-input.style.tsx

```diff
@@ -0,0 +1,91 @@
+import styled, { css } from "styled-components";
+import type { InputProps } from "../../__internal__/input";
+import type { ValidationProps } from "../../__internal__/validations";
+import StyledTypography from "../typography/typography.style";
+
+export const StyledHintText = styled.div`
```

Comment:
comment: I think we should just re-use the styled-components from https://github.com/Sage/carbon/blob/69729437c5b9bc5c9b449557ac09d203ec764033/src/components/textbox/textbox.style.ts. We should move these to a shared file but lets do that in a new PR

Reply:
yeah it's a good point, that I'm happy to raise as a ticket. I added it again here  because we already have this exported from many places (TextBox, Switch, CheckboxGroup and RadioButtonGroup) so I thought this was the standard way. Happy to tidy up to make it DRYer if we agree that's the better approach (I'm pretty sure I do btw), but it shouldn't be part of the FileInput work I don't think.

Reply:
Yeah agree not part of this work but definitely we should raise a new ticket to consolidate those (I missed that it had already been duplicated elsewhere tbh)